docs/pywbemcli/commands.rst

```diff
@@ -1331,19 +1331,55 @@ The default connection, if defined, is marked with `#` in the Name column.
+    | mock1        |                      | root/cimv2             |             |        30 |            | True     |            |           | tests/unit/simple_mock_model.mof                 |
+    +--------------+----------------------+------------------------+-------------+-----------+------------+----------+------------+-----------+--------------------------------------------------+
+    | mock1ext     |                      | root/cimv2             |             |        30 |            | True     |            |           | tests/unit/simple_mock_model_ext.mof             |
+    +--------------+----------------------+------------------------+-------------+-----------+------------+----------+------------+-----------+--------------------------------------------------+
+    | mock1interop |                      | interop                |             |        30 |            | True     |            |           | tests/unit/simple_mock_model.mof                 |
+    +--------------+----------------------+------------------------+-------------+-----------+------------+----------+------------+-----------+--------------------------------------------------+
+    | ophttp       | http://localhost     | root/cimv2             |             |        30 |            | True     |            |           |                                                  |
+    +--------------+----------------------+------------------------+-------------+-----------+------------+----------+------------+-----------+--------------------------------------------------+
+    | ophttps      | https://localhost    | root/cimv2             | blahblah    |        30 |            | False    |            |           |                                                  |
+    +--------------+----------------------+------------------------+-------------+-----------+------------+----------+------------+-----------+--------------------------------------------------+
+    | opt          | https://blah         | root/cimv2             |             |        45 |            | False    | c1.pem     | k1.pem    |                                                  |
+    +--------------+----------------------+------------------------+-------------+-----------+------------+----------+------------+-----------+--------------------------------------------------+
+    | test1        |                      | root/cimv2             |             |        30 |            | True     |            |           | tests/unit/simple_assoc_mock_model.mof           |
+    +--------------+----------------------+------------------------+-------------+-----------+------------+----------+------------+-----------+--------------------------------------------------+
+
+
+`Connection list` does not display the password or the ca-certs fields.  See
+:ref:`Connection show command` to display these fields.
```

Comment:
The "connection show" command shows the password as asterisks. Saying that "connection show" displays the password field causes security sensitive people to start sweating, and while there is a password entry shown the password information is not revealed at all. So this sentence needlessly causes a bad impression.

Why not simply omitting "the password or" from this sentence?

Reply:
Changed to:

`Connection list` does not display some fields such as the ca-certs field.  See
:ref:`pywbemcli connection show` for more detailed display of individual fields
used by the server.